client/signup/config/steps.js

```diff
@@ -190,6 +190,7 @@ export default {
 		apiRequestFunction: createAccount,
 		props: {
 			oauth2Signup: true,
+			isSocialSignupEnabled: config.isEnabled( 'signup/social' ),
```

Comment:
Are we fine with enabling Google in all OAuth signup flows? This will impact WooCommerce and other services:

Before | After
------ | -----
<img width="736" alt="before" src="https://user-images.githubusercontent.com/594356/35195207-79791a90-fec0-11e7-9745-241cd2bac852.png"> | <img width="736" alt="after" src="https://user-images.githubusercontent.com/594356/35195208-7dde7ecc-fec0-11e7-8ef0-bdec343d46da.png">